src/components/MedicationGroup.js

```diff
@@ -37,19 +37,19 @@ export default function MedicationGroup({
         <>
           <SmallButton
             right="10px"
-            top="10px"
-            color="red"
+            top="20px"
             onClick={() => handleDeleteClick(id)}
+            position="absolute"
```

Comment:
Instead of adding CSS in JSX, consider adding args like `variant` or create additional styled components base on your button.